### ISSUE TITLE
Night CI 2023-03-28: IP Release failed

### ISSUE THREAD

**Issue by weizhoublue** [spidernet-io/spiderpool#1562]
action url: https://github.com/spidernet-io/spiderpool/actions/runs/4547269824

**Comment by ty-dc** [comment]
```
app.kubernetes.io/component=spiderpool-init,app.kubernetes.io/instance=spiderpool,app.kubernetes.io/managed-by=Helm,app.kubernetes.io/name=spiderpool
ns833-12502785   pod833-12488084                                              0/1     Terminating   0          2m27s   10.219.43.2     spiderpool0328202431-worker          <none>           <none>            pod833-12488084=pod833-12488
```

**Comment by iiiceoo** [comment]
It was fixed in #1661.
